backend/models/CourseUserIsLearningModel/select/index.js

```diff
@@ -1,23 +1,44 @@
-import db from '~/db/init.js';
+import knex from '~/db/knex';
+import dayjs from 'dayjs';
 
-const select = {
-  problemsToReview: (cuilId) =>
-    db.any(
-      `
-      SELECT problem.*
-      FROM problem
-      INNER JOIN problem_user_is_learning
-      ON problem_user_is_learning.problem_id = problem.id
-      WHERE
-          problem_user_is_learning.course_user_is_learning_id = \${cuilId}
-        AND
-          problem_user_is_learning.next_due_date < now()
-        AND
-          problem_user_is_learning.if_ignored = false
-      ORDER BY problem.position, problem.created_at
-      `,
-      { cuilId }
-    )
+const selectReview = async(cuilId) => {
+  const now = dayjs();
+
+  return await knex('problem')
+    .select('problem.*')
+    .join('problem_user_is_learning', {
+      'problem_user_is_learning.problem_id': 'problem.id'
+    })
+    .where({ 
+      'problem_user_is_learning.course_user_is_learning_id': cuilId,
+      'problem_user_is_learning.if_ignored': false
+    })
+    .andWhere('problem_user_is_learning.next_due_date', '<', now)
+    .orderBy([ 
+      { column: 'position' },
+      { column: 'created_at' }
+    ]);
```

Comment:
Nice!
We used to use `sql`'s "now()", and now we use `dayjs`'s "now()", hopefully it will be ok timezone-wise.